NEWS

```diff
@@ -69,7 +69,11 @@ issues]].
 None
 
 ** Backwards incompatible changes
-None
+- The nocross opcode has been made into a prefix opcode, which means
```

Comment:
- I guess This should go in the new section for 3.13 now
- Also this should probably be mentioned in both the _New Features_ section and the _Backwards incompatible changes_ section

Reply:
I'll take care of that